numpy/ma/core.py

```diff
@@ -5211,6 +5211,8 @@ def cumprod(self, axis=None, dtype=None, out=None):
     def mean(self, axis=None, dtype=None, out=None, keepdims=np._NoValue):
         """
         Returns the average of the array elements along given axis.
+        `int32` or `uint32` intermediate and return values are used for
```

Comment:
```suggestion

        `int32` or `uint32` intermediate and return values are used for
```

Usually, the docstring has a single line description of the function and any other comments follow on the next line. So can you please add a blank line before?

Reply:
Note that this is only correct on windows, all other OSs it will be 64bit on 64bit systems.

Reply:
@Mukulikaa and @seberg, I've updated the description.